lib/spack/docs/module_file_support.rst

```diff
@@ -618,7 +618,21 @@ activated using ``spack activate``:
 The configuration file above will produce module files that will
 load their direct dependencies if the package installed depends on ``python``.
 The allowed values for the ``autoload`` statement are either ``none``,
-``direct`` or ``all``.
+``direct`` or ``all``.  The default is ``none``.
```

Comment:
Didn't you say in the other thread, that `all` actually ends up being nothing? Shouldn't we then remove `all` from here?